.gitignore

```diff
@@ -0,0 +1 @@
+discordo*
```

Comment:
Ideally, you should append `.vscode/launch.json` to this file.

```suggestion
discordo*

# Visual Studio Code
.vscode/
```

Reply:
I completely forgot about that, my bad